project-base/app/src/DataFixtures/Demo/ParameterDataFixture.php

```diff
@@ -29,12 +31,85 @@ class ParameterDataFixture extends AbstractReferenceFixture implements Dependent
     public const PARAMETER_PREFIX = 'parameter_';
     public const PARAMETER_SLIDER_WARRANTY = 'parameter_slider_warranty';
```

Comment:
those two constants should be removed as the related reference is not set

Reply:
removed as well